src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequence.tsx

```diff
@@ -18,12 +18,15 @@ import React, {
   useState,
   useEffect,
   useRef,
+  useContext,
   type FormEvent,
   type ClipboardEvent,
   type FocusEvent
 } from 'react';
 import classNames from 'classnames';
 
+import { BlastFormContext } from 'src/content/app/tools/blast/views/blast-form/BlastFormContextContainer';
```

Comment:
Let's now separate BlastFormContext into its own file. It's odd that it has to be imported from a container; it should be the other way around: the container imports the context.

Reply:
I agree its feels odd for the context to be imported from the container but at the same time it feels odd to have a separate file just for 2 lines of code for the context:
```export const BlastFormContext = React.createContext<
  SequenceValidity | undefined
>(undefined);```

What about i renamed `BlastForContextContainer` to `BlastFormContext` and it will also contains the container as a separate export. Does that sound better?

Reply:
> What about i renamed BlastForContextContainer to BlastFormContext and it will also contains the container as a separate export. Does that sound better?

Yes.